### ISSUE TITLE
_api_request hardcodes method instead of using $method

### ISSUE THREAD

**Issue by hartzell** [zurborg/libdist-zilla-plugin-github-uploadrelease-perl#1]
The method to use is passed in to _api_request ([here](https://github.com/zurborg/libdist-zilla-plugin-github-uploadrelease-perl/blob/master/lib/Dist/Zilla/Plugin/GitHub/UploadRelease.pm#L62) but the actual request uses a hardcoded `POST` [here](https://metacpan.org/source/ZURBORG/Dist-Zilla-Plugin-GitHub-UploadRelease-0.002/lib/Dist/Zilla/Plugin/GitHub/UploadRelease.pm#L52-55).

**Comment by zurborg** [comment]
Yes, `$method` is not really used, except in the logging statement. I'm going to fix it.
